Tasks/DeployAzureResourceGroup/DeployAzureResourceGroup.ps1

```diff
@@ -57,15 +57,23 @@ if($isSwitchAzureModeRequired)
 if( $action -eq "Create Or Update Resource Group" )
 {
     Create-AzureResourceGroupHelper -csmFile $csmFile -csmParametersFile $csmParametersFile -resourceGroupName $resourceGroupName -location $location -overrideParameters $overrideParameters -isSwitchAzureModeRequired $isSwitchAzureModeRequired
+    if(-not [string]::IsNullOrEmpty($outputVariable))
+    {
+        Instantiate-Environment -resourceGroupName $resourceGroupName -outputVariable $outputVariable
+    }
 }
-else
+elseif( $action -eq "SelectRG")
 {
-    Perform-Action -action $action -resourceGroupName $resourceGroupName
+    if([string]::IsNullOrEmpty($outputVariable))
+    {
+        throw (Get-LocalizedString -Key "Please provide the output variable name since you have specified the 'Select Resource Group' option.")
+    }
+    
+    Instantiate-Environment -resourceGroupName $resourceGroupName -outputVariable $outputVariable
```

Comment:
Since we don't have a drop down, should we at-least check if this resource group is present in the given subscription before you add it to output variable?